src/Native/Unix/System.Native/pal_sysctl.c

```diff
@@ -15,18 +15,17 @@
 #include "pal_safecrt.h"
 
 #include <errno.h>
-#include <memory>
 
 #include <sys/types.h>
 #include <sys/sysctl.h>
 
-extern "C" int32_t SystemNative_Sysctl(int* name, unsigned int namelen, void* value, size_t* len)
+int32_t SystemNative_Sysctl(int* name, unsigned int namelen, void* value, size_t* len)
 {
-    void* newp = nullptr;
+    void* newp = NULL;
     size_t newlen = 0;
 
 #ifdef __linux__
-    return sysctl(name, static_cast<int>(namelen), value, len, newp, newlen);
+    return sysctl(name, (int)(namelen), value, len, newp, newlen);
```

Comment:
Nit: could just be `(int)namelen`

Reply:
In a local build (`build-native.sh`) this generates an error:
```
/home/gfoidl/dotnet/corefx/src/Native/Unix/System.Native/pal_sysctl.c:27:25: error: implicit conversion changes signedness: 'unsigned int' to 'int'
      [-Werror,-Wsign-conversion]
    return sysctl(name, namelen, value, len, newp, newlen);
           ~~~~~~       ^~~~~~~
1 error generated.
System.Native/CMakeFiles/System.Native.dir/build.make:518: recipe for target 'System.Native/CMakeFiles/System.Native.dir/pal_sysctl.c.o' failed
```
So I'll keep the explicit cast.

Reply:
Ahhh, I didn't get you mean the other parenthesis -- I must have been blind or something like this. Sorry.

Leave it as is or submit a PR to fix this?

Reply:
We can leave it as is for now. Thanks.